packages/rrweb-snapshot/src/utils.ts

```diff
@@ -9,6 +9,54 @@ export function isShadowRoot(n: Node): n is ShadowRoot {
   return Boolean(host && host.shadowRoot && host.shadowRoot === n);
 }
 
+interface IsInputTypeMasked {
+  maskInputOptions: MaskInputOptions;
+  tagName: string;
+  type: string | number | boolean | null;
+}
+
+/**
+ * Check `maskInputOptions` if the element, based on tag name and `type` attribute, should be masked.
+ * If `<input>` has no `type`, default to using `type="text"`.
+ */
+function isInputTypeMasked({
```

Comment:
I'm probably missing something but why does this function not simply return a boolean? I checked usages and it seems like it's only used in if conditions anyway. I guess we wouldn't need the interface then either.

Reply:
Hmm? It does return a boolean... are the parens throwing you off? (prettier added them)

Reply:
Lol, sorry - it does of course. Yup, mistook the param type as the return type. Weekend here I come 😅